particle_filter:
    kinect_epsilon_d: 0.2 #The distance noise for kinect beacon sensing
    kinect_epsilon_a: 0.05 #The angle noise for kinect beacon sensing
    kinect_random_reading_p: 0.1 #Probability of a random reading for kinect
    kinect_missed_reading_p: 0.05 #Probability of no reading when there should be one
    kinect_min_fov_angle: -0.375 #Minimum angle (in robot coordinates) of fov
    kinect_max_fov_angle: 0.375 #Maximum angle (in robot coordinates) of fov
    lidar_epsilon_d: 0.1 #The distance noise for lidar beacon sensing
    lidar_epsilon_a: 0.05  #The angle noise for lidar beacon sensing
    lidar_random_reading_p: 0.05 #Probability of a random reading for lidar
    lidar_missed_reading_p: 0.05 #Probability of no reading when there should be one
    lidar_min_fov_angle: -2.36 #Minimum angle (in robot coordinates) of fov
    lidar_max_fov_angle: 2.36 #Maximum angle (in robot coordinates) of fov
    initial_location_sd: 0.1 # initial sd of x and y coordinates of robot location
    initial_theta_sd: 0.05 #initial sd of robot's theta
    aruco_location_sd: 0.1
    aruco_theta_sd: 0.05
    number_of_particles: 500 #How many particles we have